SEED-platform/seed
# Default null measure recommendation to True

### Any background context you want to provide?
Audit Template tool does not fill out this field. By default, it should be set to true because a person has spent time to fill out the measure meaning that it is probably recommended.

#### What's this PR do?
Defaults PropertyMeasure.recommended to True.

#### How should this be manually tested?
* `./manage.py test`
* Import the buildingsync_v2_0_bricr_workflow.xml and verify that there is one explicit False recommended measure. All the others should be true (there are two non defined entries in the updated XML)

#### What are the relevant tickets?
#2689 

#### Screenshots (if appropriate)
